test-http/src/test/cve_id.py

```diff
@@ -37,6 +36,16 @@ def test_get_cve_id_bad_org_header():
     response_contains_json(res, 'error', 'UNAUTHORIZED')
 
 
+# general user test
```

Comment:
```suggestion
```

Reply:
I'm making suggestions to remove this repeated `# general user test` comment from each test. If you want to identify these tests vs. other tests in the `cve_id.py` file I would suggest the convention used elsewhere in the code of a block comment like this:

```
# CVE-ID ENDPOINT GENERAL USER TESTS
# ==============================================================================
# these tests exercise the `/api/cve-id` endpoint acting as a general user
```

Reply:
With that being said, I'm not sure the tests in this pull request constitute general user tests. These tests add significant coverage to the CVE ID endpoint and they should definitely be included. The next two steps, which I think would constitute meeting the requirements for this pull request's corresponding issue, would be to duplicate these tests using the `org_admin_headers` and `reg_user_headers` fixtures instead of `utils.BASE_HEADERS`. Not every test will need to be duplicated, just tests where the behavior being tested differs between users.